selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -1620,6 +1638,28 @@ public void waitTextElementsActiveLine(final String expectedText) {
     return getCursorPositionFromWebElement(currentActiveElement);
   }
 
+  public Pair<Integer, Integer> getCurrentCursorPositionFromVisible(int editorIndex) {
+    int adoptedIndexOfEditor = editorIndex + 1;
+    String editorActiveLineXpath =
+        format(DEFINED_EDITOR_ACTIVE_LINE_TEMPLATE_XPATH, adoptedIndexOfEditor);
+
+    List<WebElement> positionWidget =
+        seleniumWebDriverHelper.waitPresenceOfAllElements(By.xpath(editorActiveLineXpath));
+
+    String cursorPositionText =
```

Comment:
It's not quit clear what text does it mean: line where cursor is situated, or symbol at cursor position.